emission/analysis/modelling/tour_model/evaluation_pipeline.py

```diff
@@ -128,16 +133,20 @@ def test(data,radius,low,dist_pct,kmeans):
 def main(all_users):
     radius = 100
     all_filename = []
-    for a in range(len(all_users)):
-        user = all_users[a]
+    for a, user in enumerate(all_users):
+        logging.info(f"Starting evaluation for {user}")
         df = pd.DataFrame(columns=['user','user_id','percentage of 1st round','homogeneity socre of 1st round',
                                    'percentage of 2nd round','homogeneity socre of 2nd roun','scores','lower boundary',
```

Comment:
I have a typo in this line.
Should be `homogeneity socre of 2nd round`

Reply:
go ahead and submit a commit to that

Reply:
Should I merge your PR first and then submit a commit? Or just submit the commit?